.changeset/persisted-factory-workspace-files.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/factory': minor
+---
+
+Added persisted workspace file lists for Factory threads. The Files view now keeps a thread's captured file list available after an agent run while file contents continue to load from its live sandbox.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)